### ISSUE TITLE
Failed to parse source map warning

### ISSUE THREAD

**Issue by mirkonasato** [apollographql/invariant-packages#279]
After upgrading `@apollo/client` to `3.6.0` that depends on `ts-invariant` `0.10.1` I'm seeing the following warning from the React dev server at every code change:

```
Compiled with warnings.

Failed to parse source map from '/path/to/project/node_modules/ts-invariant/src/invariant.ts' file: Error: ENOENT: no such file or directory, open '/path/to/project/node_modules/ts-invariant/src/invariant.ts'

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.

WARNING in ./node_modules/ts-invariant/lib/invariant.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
```

This is in a project using `react-scripts` `5.0.1`.

Before the upgrade I had `@apollo/client` `3.5.4` which used `ts-invariant` `0.9.4` (still the latest according to the npm registry) and there were no warnings.

**Comment by timagixe** [comment]
Hi, I faced the same warning. I have created a simple repo to reproduce it. I used CRA 5.0.1 with typescript template:
```
yarn create react-app react-app-ts --template typescript
```

And installed following dependencies of Apollo Client and GraphQL:
```
"@apollo/client": "^3.6.1",
"graphql": "^16.4.0",
```

**STR:**
1. clone https://github.com/timagixe/apollo-client-warning
2. yarn install
3. yarn start

**Comment by benjamn** [comment]
This should be fixed in `ts-invariant@0.10.2` (just published to npm). Thanks to @yanickrochon @mirkonasato @timagixe for reporting!
